Source/WebCore/workers/WorkerRunLoop.cpp

```diff
@@ -284,7 +284,7 @@ WorkerDedicatedRunLoop::RunInModeResult WorkerDedicatedRunLoop::runInMode(Worker
     Seconds timeoutDelay = Seconds::infinity();
 
 #if USE(CF)
-    CFAbsoluteTime nextCFRunLoopTimerFireDate = CFRunLoopGetNextTimerFireDate(RetainPtr { CFRunLoopGetCurrent() }.get(), kCFRunLoopDefaultMode);
```

Comment:
We should probably wrap CFRunLoopGetCurrent() in a function which ends with "Singleton".
 Otherwise, safer C++ bot will complain.

Reply:
Will fix